pkg/guard/guard.go

```diff
@@ -26,22 +30,44 @@ func NewGuard(config cfg.Config, logger mon.Logger) (*LadonGuard, error) {
 		return nil, fmt.Errorf("can not create sqlManager: %w", err)
 	}
 
+	return NewGuardWithInterfaces(sqlManager), nil
+}
+
+func NewGuardWithInterfaces(manager ladon.Manager) *LadonGuard {
 	warden := &ladon.Ladon{
-		Manager: sqlManager,
+		Manager: manager,
 	}
 
 	return &LadonGuard{
 		warden: warden,
-	}, nil
+	}
 }
 
 func (g LadonGuard) IsAllowed(request *ladon.Request) error {
 	return g.warden.IsAllowed(request)
 }
 
-func (g LadonGuard) GetPolicesBySubject(subject string) (ladon.Policies, error) {
-	pol, err := g.warden.Manager.FindPoliciesForSubject(subject)
+func (g LadonGuard) GetPolicies() (ladon.Policies, error) {
+	policies := make(ladon.Policies, 0)
+
+	for offset := int64(0); true; offset += fetchLimit {
+		pols, err := g.warden.Manager.GetAll(fetchLimit, offset)
```

Comment:
A personal preference is that we shouldn't use an infinite loop here, even though `.GetAll()` will return len==0 at some point. But local checks are more efficient.